pyproject.toml

```diff
@@ -27,17 +25,17 @@ fastapi = {version = "^0.92.0", optional = true}
 starlite = {version = "^1.51.6", optional = true}
```

Comment:
Do we need a litestar entry here?

Reply:
If you added `litestar[cli]` you could remove the `rick-click` addition as it should install it.

Reply:
Looking at how `starlite` is installed as an optional, I don't think it would always be installed so i added it explicitly.. maybe it can be done though? I have more PRs incoming that maybe we can add that into.

Reply:
There is a `litestar` branch that updates a bunch of stuff, including litestar.